book_scraper/settings.py

```diff
@@ -0,0 +1,93 @@
+# Scrapy settings for book_scraper project
+#
+# For simplicity, this file contains only settings considered important or
+# commonly used. You can find more settings consulting the documentation:
+#
+#     https://docs.scrapy.org/en/latest/topics/settings.html
+#     https://docs.scrapy.org/en/latest/topics/downloader-middleware.html
+#     https://docs.scrapy.org/en/latest/topics/spider-middleware.html
+
+BOT_NAME = "book_scraper"
+
+SPIDER_MODULES = ["book_scraper.spiders"]
+NEWSPIDER_MODULE = "book_scraper.spiders"
+
+
+# Crawl responsibly by identifying yourself (and your website) on the user-agent
+#USER_AGENT = "book_scraper (+http://www.yourdomain.com)"
```

Comment:
Consider setting a custom `USER_AGENT` to identify your scraper responsibly. This helps in avoiding blocks from websites and is a good practice.